### ISSUE TITLE
Webpack version

### ISSUE THREAD

**Issue by TheoMer** [Vheissu/aurelia-starter-node#2]
Hi @Vheissu ,

In light of recent comments made over on Aurelia/Discuss (Gitter), and I quote:

'[March 24, 2016 10:54 AM](https://gitter.im/Aurelia/Discuss?at=56f3c77027087f2a2af64978)

@timtroup We have reported the issues with SystemJS to Guy Bedford. He is interested in trying to address them. However, we think there may be some more fundamental problems that are more than just a simple "fix". For example, the performance is really bad and this may be the result of a number of practices throughout the codebase. We are trying to help in any way we can, but we didn't write that library and don't control what happens to it. So, in addition to supporting it and any improved versions, we are also trying to improve our support for Webpack and RequireJS as alternatives, both of which don't suffer from these bugs and perform massively better.
Aurelia has been designed from the beginning to support multiple loaders, so we have no hard-connection to SystemJS/JSPM in the codebase. We would like the SystemJS/JSPM issues to be resolved, however we recognized from this situation that we need to ensure that our other supported options are first class as well. Developers should be able to choose what they want but we'd also like to make a recommendation in the future of a standard toolset that is more stable, less buggy and has top notch performance. **If I were starting a new project today, I would look into our new Webpack skeleton**.'

Do you have any plans to release a webpack version of your aurelia-starter-node?

Many thanks in advance.

**Comment by Vheissu** [comment]
@TheoMer Absolutely. This is definitely in the works once I find the time. I have personally switched over to Webpack myself and it is awesome.

**Comment by TheoMer** [comment]
@Vheissu Excellent! I look forward to Webpack version of your starter-node.

**Comment by michaeljoser** [comment]
@Vheissu I would love the Webpack version too.. thanks for the hard work :)
